index_test.go

```diff
@@ -1142,7 +1143,8 @@ func TestListRepos(t *testing.T) {
 			Stats: RepoStats{
 				Shards:                     1,
 				Documents:                  4,
-				IndexBytes:                 300,
+				IndexBytes:                 261,
+				IndexMmappedBytes:          49,
```

Comment:
It seems as we are counting more than before? 261 + 49 > 300?

Reply:
Before, we did not count mmapped part at all.